src/platforms/hosted/cli.c

```diff
@@ -133,76 +134,105 @@ static void cl_help(char **argv)
 	PRINT_INFO(
 		"\n"
 		"Usage: %s [-h | -l | [-vBITMASK] [-d PATH | -P NUMBER | -s SERIAL | -c TYPE]\n"
-		"  [-n NUMBER] [-j] [-C] [-t | -T] [-e] [-p] [-R[h]] [-H] [-M STRING ...]\n"
-		"  [-E | -w | -V | -r] [-a ADDR] [-S number] [file]]\n"
+		"\t[-n NUMBER] [-j] [-C] [-t | -T] [-e] [-p] [-R[h]] [-H] [-M STRING ...]\n"
+		"\t[-f | -m] [-E | -w | -V | -r] [-a ADDR] [-S number] [file]]\n"
 		"\n"
+		"The default is to start a debug server at localhost:2000\n\n"
 		"Single-shot and verbosity options [-h | -l | -vBITMASK]:\n"
-		"\t-h              Show the version version and this help, then exit\n"
-		"\t-l              List available supported probes\n"
-		"\t-v<bitmask>     Set the output verbosity level based on some combination of:\n"
-		"\t                  1 = INFO, 2 = GDB, 4 = TARGET, 8 = PROBE, 16 = WIRE\n"
+		"\t-h, --help       Show the version and this help, then exit\n"
+		"\t-l, --list       List available supported probes\n"
+		"\t-v, --verbose    Set the output verbosity level based on some combination of:\n"
+		"\t                   1 = INFO, 2 = GDB, 4 = TARGET, 8 = PROBE, 16 = WIRE\n"
 		"\n"
 		"Probe selection arguments [-d PATH | -P NUMBER | -s SERIAL | -c TYPE]:\n"
-		"\t-d <path>       Use a serial device at the given path (Deprecated!)\n"
-		"\t-P <number>     Use the <number>th debug probe found while scanning the\n"
-		"\t                  system, see the output from list for the order\n"
-		"\t-s <serial>     Select the debug probe with the given serial number\n"
-		"\t-c <type>       Select the FTDI-based debug probe with of the given\n"
-		"\t                  type (cable)\n"
+		"\t-d, --device     <path> Use a serial device at the given path (Deprecated!)\n"
```

Comment:
These suffer the same problems as `-v<bitmask>` - think it's probably prudent to be consistent even at the cost of a small amount of information